docs/home/howto/UPGRADE.md

```diff
@@ -255,3 +255,31 @@ With the latest Epiphany version it's possible to upgrade RabbitMQ to v3.8.9.
 It requires Erlang system packages upgrade that is done automatically to v23.1.4.
 Upgrade is performed in offline mode after stopping all RabbitMQ nodes.
 [Rolling upgrade](https://www.rabbitmq.com/upgrade.html#rolling-upgrades) is not supported by Epiphany and it is advised not to use this approach when Erlang needs to be upgraded.
+
+## Kubernetes upgrade
+
+---
+**NOTE**
+
+If the cluster to be upgraded has the Istio control plane application deployed you will run into issues when trying to upgrade the cluster.
```

Comment:
This statement is written in a little bit confusing way. Probably the verb is missing.

Reply:
```suggestion
If the cluster has to be upgraded has the Istio control plane application deployed you will run into issues when trying to upgrade the cluster.
```